packages/synapse-react-client/src/components/widgets/query-filter/CombinedRangeFacetFilter.tsx

```diff
@@ -8,17 +8,17 @@ import { Range, RangeValues } from '../Range'
 import { FacetFilterHeader } from './FacetFilterHeader'
 import { RadioValuesEnum, getRadioValue, options } from './RangeFacetFilter'
 import dayjs from 'dayjs'
-import { RangeSlider } from '../RangeSlider/RangeSlider'
+import RangeSlider from '../RangeSlider/RangeSlider'
 import { useQueryContext } from '../../QueryContext'
 import { SynapseConstants } from '../../../utils'
 import { useGetQueryResultBundleWithAsyncStatus } from '../../../synapse-queries'
+import { useQueryVisualizationContext } from '../../QueryVisualizationWrapper'
 
 export type CombinedRangeFacetFilterProps = {
   facetResults: FacetColumnResultRange[]
   label: string
   columnType: ColumnType
   onChange: (range: (RadioValuesEnum | number | string | undefined)[]) => void
-  collapsed?: boolean
```

Comment:
We don't ever set `collapsed` state to `true`, so remove the prop from all of the facet control components